dev/run-all.sh

```
@@ -0,0 +1,48 @@
+
+# Trap to gracefully shutdown both
+cleanup() {
+    echo ""
+    echo "Shutting down servers gracefully..."
+
+    # Send SIGINT to process groups (negative PID = process group)
+    kill -INT -$BACKEND_PID 2>/dev/null || true
+    kill -INT -$FRONTEND_PID 2>/dev/null || true
+
+    # Wait for graceful shutdown (up to 10 seconds)
+    for i in {1..10}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)